lib/mat_SDCZ.mli

```diff
@@ -52,6 +52,15 @@ val to_array : mat -> num_type array array
 (** [to_array mat] @return an array of arrays initialized from matrix
     [mat]. *)
 
+val of_list : num_type list list -> mat
+(** [of_list ar] @return a matrix initialized from the list of lists
+    [ar]. It is assumed that the OCaml matrix is in row major order
+    (standard). *)
+
+val to_list : mat -> num_type list list
+(** [to_array mat] @return a list of lists initialized from matrix
+    [mat]. *)
```

Comment:
Improve the doc.  One does not need the comment to know that this function returns a list of list and this information should not be repeated.  What the user wants is to know how the elements of `mat` are arranged in the output.